### ISSUE TITLE
DateRange Slider ToolTip getting cut off by adjacent plot

### ISSUE THREAD

**Issue by David-Herman** [bokeh/bokeh#7117]
Hello,

I have a DateRange slider inside a widgetbox adjacent to a plot. When I slide the range slider the tool tip indicating date gets cut off. I assume a simple change in the z-index in the bokeh css would fix this. I am working with Bokeh 0.12.9

```
inputs = widgetbox(text_input1,text_input2, range_slider, text_input3)
```
```
plot_layout = layout([[[inputs,
                        plot_1,
                        plot_2,
                        plot_3],  [main_plot]]])
```
![image](https://user-images.githubusercontent.com/22305345/32109283-40c1af66-bb02-11e7-9f71-6bca6c131cf2.png)

**Comment by mattpap** [comment]
>  I assume a simple change in the z-index in the bokeh css would fix this.

Not necessarily. `z-index` works for a DOM element and its descendants, but here you have two unrelated elements. In other words, `z-index` is not an absolute value.

**Comment by mattpap** [comment]
There is a similar issue #1744, but for hover tooltips.
